tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -34,6 +34,43 @@ Given('the property {string} has been set to {string} in phoenix config file', f
   return fs.writeFileSync(this.fullPathOfConfigFile, JSON.stringify(data, null, 4))
 })
 
+function setconfig(key, subkey, value, configfile) {
+  const data = getConfigJsonContent(configfile)
+  if (!data[key]) {
+    data[key] = {}
+  }
+  data[key][subkey] = value
+  return fs.writeFileSync(configfile, JSON.stringify(data, null, 4))
+}
+
+Given('the property {string} of {string} has been set to true in phoenix config file', function(
+  subkey,
+  key
+) {
+  return setconfig(key, subkey, true, this.fullPathOfConfigFile)
+})
+
+Given('the property {string} of {string} has been set to false in phoenix config file', function(
+  subkey,
+  key
+) {
+  return setconfig(key, subkey, false, this.fullPathOfConfigFile)
+})
+
+When('the property {string} of {string} is changed to true in phoenix config file', function(
+  subkey,
+  key
+) {
+  return setconfig(key, subkey, true, this.fullPathOfConfigFile)
+})
```

Comment:
```suggestion
When('the property {string} of {string} is changed to {string} in phoenix config file', function(
  subkey,
  key,
  value
) {
  return setconfig(key, subkey, value, this.fullPathOfConfigFile)
})
```

Reply:
If we do that then we will also need to send the type of the value and that IMO would not look very nice